scaleway/docs-content
# feat(docs): vale styles creation part 1

**What does this PR do?**
- Creates a new subfolder `scw_styles` to hold all style rules that we want Vale to use for our docs
- Creates new style `yml` files in this subfolder (some created from scratch, some adapted from Microsoft/write-good styles) which enforce rules from the Scaleway Documentation Guidelines (https://confluence.infra.online.net/display/MARKETTECHANIM/Documentation+Guidelines)
- Edits `vale.ini`to tell it to use only these styles/rules

- Creates new subfolder `scw_styles_indev` to hold style rules that are in development
- These styles are for rules that are harder to implement and require additional work/research
- Finishing these styles will be done in the future in a separate PR
- Vale is not currently applying styles in this subfolder

**Supporting document with more information on Vale and the style rules we have implemented**
See: https://docs.google.com/document/d/1ptiOAhmXlvrgc_N1iV2MoNaom05v8tingbsrfqpPm08/edit

**To test**
- pull this PR and navigate to the content folder or any of its subfolders
- run `vale` + `name-of-document` eg `vale compute/instances/how-to/create-an-instance.mdx`
- don't hesitate to test the rules by altering a sample document with "bad" wording to check that Vale picks up on it
